### DISCUSSION TITLE
关于评论的楼中楼排版问题

### DISCUSSION THREAD

**Discussion by tikko-now** [Richasy/Bili.Uwp#659] [想法]
评论的楼中楼排布和层主的评论排布再相同的一列下，这样显示的层级关系没有那么明显，有些混淆，是否可以将层内的评论相对于层主，向右偏移几个字符，好突出层级感

**Comment by tikko-now** [comment]
还有一个小想法，是关于播放器页面排版问题，现在的播放器的布局对于电脑屏幕来说，是不是太高了，下方的详情页留白十分的大，对于不是带鱼屏的屏幕来说，上面的播放器就十分狭长，尤其是播放16比9的视频来说，播放器左右又会留黑，有一些些不太协调。是否可以压缩一下下方的详情区域扩大一下部分器布局呢？
![QQ截图20211125120815](https://user-images.githubusercontent.com/94343527/143378246-7df1540c-6050-4648-a2a8-69707ddb9482.png)

**Reply by Q-Tai-mu** [reply]
这些空白区是打广告用的
